Tasks/UseDotNetV2/usedotnet.ts

```diff
@@ -0,0 +1,39 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+import { DotnetCoreInstaller} from './installer';
+import * as proxyutil from './proxyutil';
+import * as path from 'path';
+
+async function run() {
+    // set the console code page to "UTF-8"
```

Comment:
this was in the old code before they called some specific tool, correct? do we still call that?

Reply:
No, I got rid of this accordingly.